includes/wikia/services/CommunityDataService.class.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+class CommunityDataService extends WikiaService {
+	const CURATED_CONTENT_VAR_NAME = 'wgWikiaCuratedContent';
+	private $curatedContentData = [ ];
+
+	private $cityId;
+
+	function __construct( $cityId ) {
+		parent::__construct();
+		$this->cityId = $cityId;
+	}
+
+	public function setCuratedContent( $data ) {
+		$ready = $this->isOldFormat( $data ) ? $this->toNew( $data ) : $data;
+		$status = WikiFactory::setVarByName( self::CURATED_CONTENT_VAR_NAME, $this->cityId, $ready );
+		if ( $status ) {
+			wfWaitForSlaves();
+		}
+		return $status;
+	}
+
+	public function getCuratedContent( $useLegacyFormat = false ) {
```

Comment:
Can we make it an enum with two possible values, i.e. 
FORMAT_STRUCTURED and FORMAT_LEGACY ?